scripts/pre-push

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# INSTRUCTIONS:
+# This script will run the pre-push checks in the given order
+# Ktlint
+#
+
+# this will run the ktlint check script
```

Comment:
Comment seems unnecessary since the line is fairly self-explanatory.

Reply:
done